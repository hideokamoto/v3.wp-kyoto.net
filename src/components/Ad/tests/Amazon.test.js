/* eslint-disable react/jsx-filename-extension */
import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import AdAmazonLeaderBoard from "../Amazon";
configure({ adapter: new Adapter() });

describe("AdAmazonLeaderBoard", () => {
  it("should match snapshot", () => {
    const wrapper = shallow(<AdAmazonLeaderBoard />);
    expect(wrapper).toMatchSnapshot();
  });
});
