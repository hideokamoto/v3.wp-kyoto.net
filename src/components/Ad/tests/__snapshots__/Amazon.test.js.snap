// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdAmazonLeaderBoard should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AdAmazonLeaderBoard />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "centered": true,
      "children": <iframe
        height="90"
        scrolling="no"
        src="https://rcm-fe.amazon-adsystem.com/e/cm?f=ifr&t=wp-kyoto-22&o=9&p=48&l=st1&mode=kindle-jp&search=WordPress&fc1=000000&lt1=_top&lc1=A43907&bg1=FFFFFF"
        width="728"
      />,
      "unit": "leaderboard",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "height": "90",
        "scrolling": "no",
        "src": "https://rcm-fe.amazon-adsystem.com/e/cm?f=ifr&t=wp-kyoto-22&o=9&p=48&l=st1&mode=kindle-jp&search=WordPress&fc1=000000&lt1=_top&lc1=A43907&bg1=FFFFFF",
        "width": "728",
      },
      "ref": null,
      "rendered": null,
      "type": "iframe",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "centered": true,
        "children": <iframe
          height="90"
          scrolling="no"
          src="https://rcm-fe.amazon-adsystem.com/e/cm?f=ifr&t=wp-kyoto-22&o=9&p=48&l=st1&mode=kindle-jp&search=WordPress&fc1=000000&lt1=_top&lc1=A43907&bg1=FFFFFF"
          width="728"
        />,
        "unit": "leaderboard",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "height": "90",
          "scrolling": "no",
          "src": "https://rcm-fe.amazon-adsystem.com/e/cm?f=ifr&t=wp-kyoto-22&o=9&p=48&l=st1&mode=kindle-jp&search=WordPress&fc1=000000&lt1=_top&lc1=A43907&bg1=FFFFFF",
          "width": "728",
        },
        "ref": null,
        "rendered": null,
        "type": "iframe",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
